config/environments/development.rb

```diff
@@ -61,4 +61,13 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.action_mailer.delivery_method = :smtp
+    config.action_mailer.smtp_settings = {
+      :address => "smtp.gmail.com",
+      :port => 587,
+      :user_name =>"santiagoguzmanarango@gmail.com",
+      :password => "90051468065",
+      :authentication => "plain",
+      :enable_starttls_auto => true
+    }
```

Comment:
This configuration must be placed on environment files like `development.rb`  and `production.rb`